web_trackers/README.md

```diff
@@ -9,6 +9,7 @@ It will connects to Snowplow Mini, which is used for development and testing. On
 * [`gov_search_sp_script.js`](./gov_search_sp_script.js) should only be used for Search pages in CMS Lite.
 * [`wordpress_sp_script.js`](./wordpress_sp_script.js) should be used by sites using the standard Wordpress setup. This includes Engage.gov sites and Standalone sites. 
 * [`Snowplow_inline_code.js`](./Snowplow_inline_code.js) can be placed on any Standalone site that uses GDX Analytics services.
+* [`Snowplow_inline_code_search.js`](./Snowplow_inline_code_search.js) can be placed on any Standalone site that uses GDX Analytics services and has a search fuction. Please change the parameter in the if statment called 'search term parameter' to what is being used as the search parameter on your site. In most cases it is 'q' and for wordpress it is 's'.
```

Comment:
Here you need to let them know to change `search term parameter` in _two_ places. The line numbers are 22 and 24, if that's useful to point out instead of "in the if statment" (also, spelling error in statement).

Reply:
If it needs to be changed in two places, we should set a variable in one place and use it in both places. 

Requiring a duplicated customization is a recipe for buggy code.